src/lib/core/_core.scss

```diff
@@ -48,7 +49,8 @@
   }
 
   // Marker that is used to determine whether the user has added a theme to their page.
-  .mat-theme-loaded-marker {
+  .mat-theme-loaded-marker,
+  &.mat-theme-loaded-marker {
```

Comment:
I'm not sure whether it makes sense to change the selector on this one. The class gets applied to an element that we create dynamically which gets removed immediately afterwards.